website/src/pages/Blog/index.tsx

```diff
@@ -190,6 +193,7 @@ const Blog: React.FC = () => {
 
   return (
     <>
+     <Metadata title={SeoJson.pages.blog.title} description={SeoJson.pages.blog.description} url={`${getHostOrigin}${SeoJson.pages.blog.url}`} image={`${getHostOrigin}${SeoJson.pages.blog.image}`} isPost={false} type="Series"  />
```

Comment:
```suggestion
     <Metadata title={SeoJson.pages.blog.title} description={SeoJson.pages.blog.description} url={`${getHostOrigin}${SeoJson.pages.blog.url}`} image={`${getHostOrigin}${SeoJson.pages.blog.image}`} isPost={false} type="Article"  />
```
Type should be article right?

Reply:
A `Series` in schema.org is a group of related items
https://schema.org/Series